modules/openy_features/openy_node/openy_node.features.yml

```diff
@@ -1 +1,7 @@
+excluded:
+  - core.entity_form_display.taxonomy_term.color.default
+  - core.entity_view_display.taxonomy_term.color.default
+  - field.field.taxonomy_term.color.field_color
+  - field.storage.taxonomy_term.field_color
+  - taxonomy.vocabulary.color
```

Comment:
We should remove this from here, because in the installation process they will know nothing about each other.